proto/tendermint/types/params.proto

```diff
@@ -56,7 +56,7 @@ message ValidatorParams {
   option (gogoproto.populate) = true;
   option (gogoproto.equal)    = true;
 
-  repeated string pub_key_types = 1;
+  string pub_key_type = 1;
```

Comment:
Another type breakage here.

Reply:
can revert and assumer the length will always be 1

Reply:
reverted